src/c++/perf_analyzer/perf_analyzer.cc

```diff
@@ -52,6 +52,44 @@ SignalHandler(int signum)
     exit(0);
   }
 }
+
+bool
+IsLLMModel(
+    const std::shared_ptr<pa::ModelParser>& parser,
+    const pa::PAParamsPTR& params)
+{
+  bool is_llm_from_user = params->is_llm if (is_llm_from_user)
+  {
+    return true;
+  }
+
+  bool is_llm = false;
+  // check if its decoupled
+  is_llm =
+      is_llm || (parser->IsDecoupled() && !params->profile_export_file.empty());
+
+  // check if is ensemble model, and if model has a tensorrt_llm portion to it
+  // then it is for sure the tensorrt-llm backend
+  if (!parser->composing_models_map_.empty()) {
```

Comment:
Does `composing_models_map_` get populated for BLS models too if they provide the `--bls-composing-models`?